vpietri/magento2-developer-quickdevbar
# [FEATURE] Add option allow disable toolbar from admin area

### Description (*)

Add an option allow to disable dev toolbar from admin area only

### Fixed Issues (if relevant)
1. [vpietri/magento2-developer-quickdevbar/#26](https://github.com/vpietri/magento2-developer-quickdevbar/issues/26):  Disable toolbar in Admin 

### Manual testing scenarios (*)
1.  Go Stores/Configuration/Advanced/Developer/Quick dev bar configuration 

2. Change Disable Dev Bar from Admin Area: Yes - No

3. See the Dev bar disabled from admin or not. Frontend devbar has no affect with this configuration
